ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -239,4 +239,100 @@ public static synchronized String getSchemaDefinitionString(Schema schema) {
     return stringBuilder.toString();
   }
 
+  public static String getSQLTypeName(Schema schema) {
+    switch (schema.type()) {
+      case INT32:
+        return "INT";
+      case INT64:
+        return "BIGINT";
+      case FLOAT32:
+      case FLOAT64:
+        return "DOUBLE";
+      case BOOLEAN:
+        return "BOOLEAN";
+      case STRING:
+        return "VARCHAR";
+      case ARRAY:
+        return "ARRAY<" + getSQLTypeName(schema.valueSchema()) + ">";
+      case MAP:
+        return "MAP<" + getSQLTypeName(schema.keySchema()) + "," + getSQLTypeName(schema
+                                                                                       .valueSchema()) + ">";
+      default:
+        throw new KsqlException(String.format("Invalid type in schema: %s.", schema.toString()));
+    }
+  }
+
+  public static  synchronized String buildAvroSchema(final Schema schema, String name) {
```

Comment:
Why synchronized?

Reply:
Also, it looks like this method is building an Avro schema string from a connect schema object.

Why can't you use the avro schema builder and then generate the string from that? It would be far cleaner and less error prone.

Reply:
I switched to Avro Schema builder to build the json string.